numba_cuda/numba/cuda/codegen.py

```diff
@@ -246,10 +246,11 @@ def get_cufunc(self):
             return cufunc
 
         cubin = self.get_cubin(cc=device.compute_capability)
-        module = ctx.create_module_image(cubin)
+        #module = ctx.create_module_image(cubin)
 
         # Load
-        cufunc = module.get_function(self._entry_name)
+        #cufunc = module.get_function(self._entry_name)
+        cufunc = cubin.get_kernel(self._entry_name)
```

Comment:
Since we switched to context independent loading API, the `CUDADispatcher.bind` method should probably be renamed since it no longer binds context via calling the `get_cufunc` function.